CHANGES.md

```diff
@@ -1,4 +1,5 @@
 ## Changelog
+  * Feature: Add stop callback when exiting loop, add stop method placeholder for base callbacks
```

Comment:
move this under the current dev version, 1.6.2

Reply:
done.